sdk/python/feast/feature_store.py

```diff
@@ -1894,6 +1894,9 @@ def serve(
         type_: str = "http",
         no_access_log: bool = True,
         workers: int = 1,
+        threads: int = 1,
+        max_requests: int = 0,
```

Comment:
how the server behaves if someone won't set this field and defaults to 0.

Reply:
@lokeshrangineni 

if `threads` is set to zero(`0`),  the only one thread is run in server.(At reference, `workers` can be set to zero. but this is only running a master process not worker process. so can't process client's request. I test it in FastAPI in localhost)

And if `max_requests` is set to zero(`0`), the automatic restart process is disabled(ref: [gunicorn docs](https://docs.gunicorn.org/en/stable/settings.html#max-requests))